definitions/procedures/content/switchover.rb

```diff
@@ -0,0 +1,18 @@
+module Procedures::Content
+  class Switchover < ForemanMaintain::Procedure
+    metadata do
+      description 'Switch support for certain content from Pulp 2 to Pulp 3'
+      for_feature :pulpcore
+    end
+
+    def run
+      execute!('foreman-rake katello:pulp3_migration')
+      execute!('foreman-rake katello:pulp3_post_migration_check')
+      execute!('foreman-rake katello:pulp3_content_switchover')
+      args = %( --foreman-proxy-content-proxy-pulp-isos-to-pulpcore=true
+                 --katello-use-pulp-2-for-file=false
+                 --katello-use-pulp-2-for-docker=false)
```

Comment:
I'm getting: 
sh: line 2: --katello-use-pulp-2-for-docker=false: command not found

it seems like those all need to be on the same line, or probably use \  at the end?

Reply:
@jlsherrill thanks, I just moved it to an array of strings that is joined so its still easy to read and we don't have to deal with multi-line ruby strings and compatibility, indentation, + new lines.